overlays/haskell.nix

```diff
@@ -540,7 +494,6 @@ final: prev: {
                       evalPackages = final.lib.mkDefault evalPackages;
                       inputMap = final.lib.mkDefault inputMap;
                     } ];
-                  extra-hackages = args.extra-hackages or [] ++ callProjectResults.extra-hackages;
```

Comment:
did you purge this from everywhere? since now it seems like it's totally unsupported